cinema/urls.py

```diff
@@ -1 +1,23 @@
-# write urls here
+from django.urls import path, include
+from rest_framework import routers
+
+from .views import (
+    GenreViewSet,
+    CinemaHallViewSet,
+    MovieSessionViewSet,
+    MovieViewSet,
+    ActorViewSet
+)
+
+router = routers.DefaultRouter()
+router.register("genres", GenreViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+router.register("movies", MovieViewSet)
+router.register("actors", ActorViewSet)
+
+urlpatterns = [
+    path("", include(router.urls))
+]
+
```

Comment:
No need to include if base path is not provided you can do:
urlpatters = router.urls